contrail-agent/hooks/contrail_agent_hooks.py

```diff
@@ -126,27 +109,15 @@ def install():
     os.chmod("/etc/contrail", 0o755)
     os.chown("/etc/contrail", 0, 0)
 
-    if config.get("dpdk"):
-        install_dpdk()
-    else:
+    if not config.get("dpdk") and not init_is_systemd():
         # supervisord must be started after installation
-        if not init_is_systemd():
-            # supervisord
-            service_restart("supervisor-vrouter")
-        install_dkms()
-        if config.get("agilio-vrouter"):
-            install_agilio()
-
-def install_agilio():
-    configure_virtioforwarder()
-    service("enable","virtio-forwarder")
-    service("start","virtio-forwarder")
-    configure_apparmor()
-    iface = config.get("physical-interface")
-    check_call("ifdown " + iface, shell=True)
-    configure_initramfs()
-    check_call("ifup " + iface, shell=True)
-    check_call("ifup vhost0", shell=True)
+        service_restart("supervisor-vrouter")
+
+    lazy_install()
+    config["vrouter-expected-provision-state"] = False
```

Comment:
my fail - lazy_install must be after config operations:

config["vrouter-expected-provision-state"] = False
config["vhost-ready"] = False
config.save()
lazy_install()

in current case without plugin - lazy_install will set second flag to True and this code will be reset to False